marketplace/mux-video/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "mux-contentful-uploader",
+  "version": "1.0.0",
+  "private": true,
+  "devDependencies": {
+    "@babel/core": "^7.4.0",
```

Comment:
By default we use `7.3.4` of this dependency with current `.babelrc` setup. With `7.4.0` I see the following warning during `npm run start`:

```bash
WARNING: We noticed you're using the `useBuiltIns` option without declaring a core-js version. Currently, we assume version 2.x when no version is passed. Since this default version will likely change in future versions of Babel, we recommend explicitly setting the core-js version you are using via the `corejs` option.

You should also be sure that the version you pass to the `corejs` option matches the version specified in your `package.json`'s `dependencies` section. If it doesn't, you need to run one of the following commands:

  npm install --save core-js@2    npm install --save core-js@3
  yarn add core-js@2              yarn add core-js@3
```

Would be nice to address the issue

Reply:
I actually updated in order to try and _fix_ this issue (it did not). I downgraded but still see the warning, so let me know how you want to proceed.

Reply:
I think all of the babel dependencies should match `7.3.*` to remove the warning.